misc/helm-charts/operator/templates/deployment.yaml

```diff
@@ -103,6 +103,12 @@ spec:
         {{ if .Values.operator.clusters.defaultSizes.analytics }}
         - "--bootstrap-builtin-analytics-cluster-replica-size={{ .Values.operator.clusters.defaultSizes.analytics }}"
         {{- end }}
+        {{ if .Values.operator.clusters.defaultReplicationFactor.system }}
+        - "--bootstrap-builtin-system-cluster-replication-factor={{ .Values.operator.clusters.defaultReplicationFactor.system }}"
+        {{- end }}
+        {{ if .Values.operator.clusters.defaultReplicationFactor.probe }}
+        - "--bootstrap-builtin-probe-cluster-replication-factor={{ .Values.operator.clusters.defaultReplicationFactor.probe }}"
+        {{- end }}
```

Comment:
@jubrad wasn't sure if I should include all the clusters, or just these two for orchestratord. Also wasn't sure if we should get rid of the `mz_probe` size and a bunch of the other default sizes?